docs/docusaurus/docs/cloud/deploy_gx_agent.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: 'Deploy the GX Agent'
+title: 'Deploy the GX Agent'
+id: deploy_gx_agent
+description: Deploy the GX Agent to use GX Cloud features and functionality.
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To use GX Cloud features and functionality, you need to deploy the GX Agent. The GX Agent is an intermediary between GX Cloud and your organization's data stores. GX Cloud does not connect directly to your data, all data access occurs within the GX Agent. GX Cloud sends jobs to the GX Agent, the GX Agent executes these jobs against your data, and then sends the job results to GX Cloud.
+
+The GX Agent is typically deployed in your organization's deployment environment, for example, in a development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows. To learn more about the GX Agent and deployment patterns, see [About GX Cloud](./about_gx.md).
+
+## Prerequisites
+
+- You have a [GX Cloud account](https://greatexpectations.io/cloud).
+
+## Get your access token and organization ID
+
+You need your access token and organization ID to deploy the GX Agent. Access tokens shouldn't be committed to version control software. 
+
+If you've used GX Cloud previously, you have your access token and organization ID, and you need to restart the GX Agent, see [Deploy the GX Agent](#deploy-the-gx-agent).
+
+<Tabs
+  groupId="copy-token"
+  defaultValue='new'
+  values={[
+  {label: 'New GX Cloud account', value:'new'},
+  {label: 'Existing GX Cloud account', value:'existing'},
+  ]}>
+<TabItem value="new">
+
+1. Sign in to GX Cloud.
+
+2. Complete the survey and then click **Continue to GX Cloud**.
+
+3. Copy and then paste the **Access token** and **Organization ID** values into a temporary file. You'll need them to deploy the GX Agent.
+
+4. Click **Deploy the GX Agent** and [deploy the GX Agent](#deploy-the-gx-agent).
```

Comment:
Is Step 4 here necessary? This section is just about getting your token and org ID.

Reply:
@annabarr that's part of the process. Users need to know what to do when they have recorded the access token and organization ID.

Reply:
Ok that works for me